### ISSUE TITLE
Request for training script for MAI-UI

### ISSUE THREAD

**Issue by jangyicheng** [ZJU-REAL/ClawGUI#7]
Thank you for your great work! I noticed that the `clawgui-rl/examples/gigpo_trainer` directory only contains an example of a training script, not the training script for MAI-UI mentioned in the paper. Could you please provide the training script for MAI-UI-2B? Thank you very much!

**Comment by sugarandgugu** [comment]
Hi, thanks for your interest! You can adapt the script with the following key changes:

CUDA_VISIBLE_DEVICES=0,1,2,3,4,5,6,7
model_type=maiui
n_gpus=8
history_length=3
max_steps=50
train_data_size and group_size=8

Also, when launching the environment, you can start multiple instances as spare servers. For the reward PRM, we use Qwen 3.5 27B.
